RestaurantReservation/Services/CustomerService.cs

```diff
@@ -0,0 +1,57 @@
+﻿using RestaurantReservation.Db.Models;
+using RestaurantReservation.Db;
+using Microsoft.EntityFrameworkCore;
+namespace RestaurantReservation.Services
```

Comment:
We used to use repositories for DB operations, and services if we wanted to add logic  for (read/ write operations)